workflow_tests/test_verified_translate.py

```diff
@@ -0,0 +1,100 @@
+from dotenv import load_dotenv
+
+import os
+import pytest
+import lilt
+
+from tenacity import (
+    retry,
+    stop_after_delay,
+    wait_exponential,
+    retry_if_result,
+    RetryError,
+)
+
+load_dotenv()
+
+
+@pytest.fixture(scope="module")
+def client():
+    configuration = lilt.Configuration(
+        host=os.environ["API_HOST"], api_key={"key": os.environ["API_KEY"]}
+    )
+    api_client = lilt.ApiClient(configuration)
+    commit = os.environ.get("GIT_COMMIT_SHA", "no_version_available")
+    api_client.user_agent = f"lilt-python-e2e-tests/{commit}"
+    return api_client
+
+
+@pytest.fixture(scope="module")
+def api(client):
+    return lilt.JobsApi(client)
+
+
+@pytest.fixture(scope="module")
+def uploaded_file(client):
+    api = lilt.FilesApi(client)
+    name = "test_file.txt"
+    body = b"hello world"
+    response = api.upload_file(name, body)
+    yield response
+    api.delete_file(response.id)
```

Comment:
fyi @nicholas-chiu this is a pattern you can use to auto clean up resources you create